uxarray/core/dataarray.py

```diff
@@ -403,9 +421,15 @@ def to_raster(
                         + input_ax_attrs._value_comparison_message(pm_ax_attrs)
                     )
             pixel_mapping = np.asarray(pixel_mapping, dtype=INT_DTYPE)
+        elif auto_extent_applied:
+            warn(
+                "Axes extent was default; auto-setting from grid lon/lat bounds for rasterization. "
+                "Set the extent explicitly to control this.",
+                stacklevel=2,
```

Comment:
This warning is very helpful! Maybe just one thing to add to make it even further useful: Elaborate on how to set extent, i.e. ""Set the extent explicitly to control this, e.g. by calling either of matplotlib's `ax.get_global()`, or `ax.set_extent()`, or `ax.set_xlim()` & `ax.set_ylim()` pair.""